src/layouts/components/Homepage/AnnouncementBody.tsx

```diff
@@ -0,0 +1,224 @@
+import { Text, Box, FormControl, VStack } from "@chakra-ui/react"
+import { DragDropContext } from "@hello-pangea/dnd"
+import {
+  FormLabel,
+  Input,
+  FormErrorMessage,
+  Button,
+  DatePicker,
+  Textarea,
+} from "@opengovsg/design-system-react"
+import _ from "lodash"
+import { BiPlus } from "react-icons/bi"
+
+import { useEditableContext } from "contexts/EditableContext"
+
+import { Editable } from "layouts/components/Editable"
+
+import { AnnouncementError, AnnouncementOption } from "types/homepage"
+
+const MAX_ANNOUNCEMENTS = 5
+
+interface AnnouncementBodyProps {
+  errors: {
+    announcementItems: AnnouncementError[]
+  }
+  announcementItems: Partial<AnnouncementOption>[]
+}
+
+export const AnnouncementBody = ({
+  errors,
+  announcementItems = [],
+}: AnnouncementBodyProps) => {
+  const {
+    onDragEnd,
+    onChange,
+    onCreate,
+    onDelete,
+    onDisplay,
+  } = useEditableContext()
+  return (
+    <Box w="full">
+      <DragDropContext onDragEnd={onDragEnd}>
+        <Editable.Droppable editableId="announcement">
+          <VStack spacing="1.25rem" align="flex-start" p={0} />
+          <Text mt="1.5rem" textStyle="h6">
+            Announcements
+          </Text>
+          <Text mt="0.5rem" textStyle="body-2" textColor="base.content.medium">
+            You can display up to 5 announcements at a time. Newly added
+            announcements are shown on the top of the list
```

Comment:
Can we use `MAX_ANNOUNCEMENTS` instead of "5" here?

Reply:
good idea 2c733f2cb227aed0b72daf0043228d64a01454cd